src/server/schema/collection.schema.ts

```diff
@@ -200,3 +209,8 @@ export const getAllCollectionsInfiniteSchema = infiniteQuerySchema
   })
   .merge(userPreferencesSchema)
   .partial();
+
+export type GetCollectionPermissionDetails = z.infer<typeof getCollectionPermissionDetails>;
+export const getCollectionPermissionDetails = z.object({
+  ids: z.array(z.number()),
```

Comment:
Probably want some constraints here
```suggestion
  ids: z.array(z.number()).min(1),
```